client/src/views/components/CreateProductSlide/ImageContainer/ImageCardList/ImageCardList.js

```diff
@@ -11,7 +11,7 @@ export default function ImageCardList({ $target, onDeleteImageHandler, imageUrls
   $target.appendChild(this.$imageCardList);
   this.$imageCardList.addEventListener('click', (e) => {
     const deleteButton = e.target.closest('button');
-    onDeleteImageHandler(deleteButton.dataset.index);
+    deleteButton && onDeleteImageHandler(deleteButton.dataset.index);
```

Comment:
취향차이일 수 있겠지만, 저였으면 early return case로 반환했을 것 같네요 :)

```suggestion
    if(!deleteButton) return;
    onDeleteImageHandler(deleteButton.dataset.index);
```

Reply:
early return도 가능하겠군요! :)
예시 들어주셔서 감사합니다!

Reply:
이 부분은 deleteButton을 한번 검사하고 넘어가는 게 똑같아서, 
한줄로 끝내는 코드로 냅둬도 괜찮을까요??!

Reply:
그럼요 :)